src/main/kotlin/com/openlattice/organization/OrganizationExternalDatabaseColumn.kt

```diff
@@ -0,0 +1,54 @@
+package com.openlattice.organization
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.authorization.securable.AbstractSecurableObject
+import com.openlattice.authorization.securable.SecurableObjectType
+import com.openlattice.client.serialization.SerializationConstants
+import java.util.*
+
+/**
+ * Creates a securable object for an organization's entire database
+ *
+ * @param id An optional UUID that will be automatically generated if not provided
+ * @param name The name of the column
+ * @param title A title for the object
+ * @param description A description for the object
+ * @param tableId The id of the table that contains this column
+ * @param organizationId The id of the organization that owns this column
+ * @param dataType The sql data type of this column
+ * @param isPrimaryKey A boolean denoting if the column is a primary key of the containing table
+ * @param ordinalPosition The index of the column within the containing table
+ */
+
+class OrganizationExternalDatabaseColumn
+
+constructor(
+        @JsonProperty(SerializationConstants.ID_FIELD) id: Optional<UUID>,
+        @JsonProperty(SerializationConstants.NAME_FIELD) var name: String,
+        @JsonProperty(SerializationConstants.TITLE_FIELD) title: String,
+        @JsonProperty(SerializationConstants.DESCRIPTION_FIELD) description: Optional<String>,
+        @JsonProperty(SerializationConstants.TABLE_ID) var tableId: UUID,
+        @JsonProperty(SerializationConstants.ORGANIZATION_ID) var organizationId: UUID,
+        @JsonProperty(SerializationConstants.DATATYPE_FIELD) var dataType: String,
+        @JsonProperty(SerializationConstants.IS_PRIMARY_KEY) var isPrimaryKey: Boolean,
```

Comment:
I'd double check behavior on `isPrimaryKey` -- I've seen some strange jackson serialization issues on kotlin data classes where the @JsonProperty name is `isBlah` and the variable name is `isBlah` (seen just `blah` get serialized due to the built in getters)

https://github.com/FasterXML/jackson-module-kotlin/issues/52

Reply:
Better to be safe, I'll just change it